examples/inverse/plot_mixed_norm_inverse.py

```diff
@@ -80,13 +70,13 @@
 idx = np.argmax([np.max(np.abs(dip.amplitude)) for dip in dipoles])
 plot_dipole_locations(dipoles[idx], forward['mri_head_t'], 'sample',
                       subjects_dir=subjects_dir, mode='orthoview',
-                      idx='amplitude')
+                      idx='amplitude', coord_frame='head')
```

Comment:
why do you plot in head? it makes the MRI clipped and it's harder to see where the dipole is with a tilted brain

Reply:
Mostly just to see it work properly, will revert